lib/Sema/ResilienceDiagnostics.cpp

```diff
@@ -180,18 +182,23 @@ diagnoseGenericArgumentsExportability(SourceLoc loc,
 
     SubstitutionMap subConformanceSubs =
         concreteConf->getSubstitutions(userSF.getParentModule());
-    diagnoseGenericArgumentsExportability(loc, subConformanceSubs, userSF);
+    diagnoseGenericArgumentsExportability(loc, subConformanceSubs, userSF, userDC);
 
     const RootProtocolConformance *rootConf =
         concreteConf->getRootConformance();
     ModuleDecl *M = rootConf->getDeclContext()->getParentModule();
-    if (!userSF.isImportedImplementationOnly(M))
+
+    auto originKind = getDisallowedOriginKind(
+        rootConf->getDeclContext()->getInnermostDeclarationDeclContext(),
```

Comment:
getInnermostDeclarationDeclContext() should be a no-op here since a conformance DeclContext is always either a NominalTypeDecl or an ExtensionDecl.

Reply:
Good to know, thanks!